alibi_detect/cd/tensorflow/context_aware.py

```diff
@@ -4,13 +4,36 @@
 import tensorflow_probability as tfp
 from typing import Callable, Dict, Optional, Tuple, Union, List
 from alibi_detect.cd.base import BaseContextMMDDrift
-from alibi_detect.utils.tensorflow.kernels import GaussianRBF
+from alibi_detect.utils.tensorflow.kernels import GaussianRBF, BaseKernel
 from alibi_detect.cd._domain_clf import _SVCDomainClf
 from tqdm import tqdm
 
 logger = logging.getLogger(__name__)
 
 
+def _sigma_median_diag(x: tf.Tensor, y: tf.Tensor, dist: tf.Tensor) -> tf.Tensor:
```

Comment:
Why is this moved?

Reply:
Probably due to a linter error of an undefined function if my memory is correct.